.travis.yml

```diff
@@ -44,15 +44,15 @@ script:
   - echo "Working copy disk usage"; du -chs .
   - grunt --version
   - grunt help
-  - grunt --quiet --timer
```

Comment:
The package command had a bug that was not completing all of its tasks, so without seeing the task output it was difficult to debug.  Also hard to see the actual tasks being run through the cloud of all the other output generated in the travis log.  I don't consider those desktop notifications irrelevant for travis as they aided me in debugging.  And it doesn't hurt to have them, right?